pinterest-for-woocommerce.php

```diff
@@ -140,6 +140,7 @@ function () {
 	function () {
 		Automattic\WooCommerce\Pinterest\ProductSync::cancel_jobs();
 		Automattic\WooCommerce\Pinterest\Heartbeat::cancel_jobs();
+		Automattic\WooCommerce\Pinterest\FeedGenerator::deregister();
```

Comment:
```suggestion
		Automattic\WooCommerce\Pinterest\ProductSync::deregister();
```
Should we use `ProductSync::deregister` instead? it in turns [calls the above and also calls](https://github.com/woocommerce/pinterest-for-woocommerce/blob/a0f59bb4c41753e58cb8ffa5ecadd56773ea7a9e/src/ProductSync.php#L207) `deregister` on a couple other items that may be relevant.

Suggesting this, as I did briefly see the `Successfully generated 5 minutes ago` message with the link to the original XML file, although it hadn't generated the new one yet and it was still pointing at the deleted version.

Reply:
The method was `private`, but I've updated it to `public` and implemented it since it does seem more comprehensive: f256a4bcd086204bd5692e0ac44b14a47cba4b31